.github/workflows/build.yml

```diff
@@ -297,7 +297,7 @@ jobs:
         uses: dtolnay/rust-toolchain@nightly
         with:
           components: clippy
-      - run: cargo clippy --package rustls --all-features
-      - run: cargo clippy --package rustls --no-default-features -- --deny warnings
-      - run: cargo clippy --manifest-path=connect-tests/Cargo.toml --all-features -- --deny warnings
-      - run: cargo clippy --manifest-path=fuzz/Cargo.toml --all-features -- --deny warnings
+      - run: cargo clippy --package rustls --all-features --all-targets
```

Comment:
hm, why is this first one missing `--deny warnings`? (it was before too, so not a change being made in this PR)

Reply:
That difference seems to have always been there:

https://github.com/rustls/rustls/commit/66abf71bdb4efcf395e27beb2282449a6895fe56

I think my rationale would have been that we only need to `--deny warnings` on stable, failing CI for clippy issues that are new on nightly would be too painful. But then that wasn't followed when @cpu added the other nightly clippy runs.

Reply:
I think just simple oversight on my part, not anything intentional.